modules/ROOT/pages/admin-console-tutorial.adoc

```diff
@@ -1,4 +1,6 @@
 = Pulsar Admin Console tutorial
+:luna-version: 2.8
+:pulsar-version: 2.8.1
```

Comment:
Let's add these variables to the antora.yaml file so they can be updated in one place.

asciidoc:
  attributes:
    luna-version: '2.8'
    pulsar-version: '2.8.1'